File: Python/exercicio-05/solucao.py
def calcular_media():
    prova1 = int(input('Qual foi a nota da prova 1 ?'))
    prova2 = int(input('Qual foi a nota da prova 2 ?'))
    media = (prova1 + prova2) / 2
    print(f'A média entre {prova1} e {prova2} é {media}')
    input('Pressione Enter Para Sair...')


calcular_media()

File: JavaScript/exercicio-05/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function media() {
  const prova1 = parseFloat(prompt('Digite a nota da prova 1 '));
  const prova2 = parseFloat(prompt('Digite a nota da prova 2 '));
  const media = (prova1 + prova2) / 2

  console.log(`A média entre ${prova1} e ${prova2} é ${media}`)
  readline.question('Digite enter para sair ... ')
}

media();